packages/documentation/src/content/docs/integration/requirements/peers.mdx

```diff
@@ -0,0 +1,287 @@
+
+After adding a peer, we need to create a wallet address. This wallet address is where payments are sent from and received. To learn how to create a new wallet address, refer to [Assets](/integration/requirements/wallet-addresses/).
+
+## Managing peers
+
+Once a peer has been added to your Rafiki instance, ongoing peer management ensures that your peering integration remains functional. This includes editing configurations to reflect changes in your peer's technical details or resolving setup errors. It also includes deleting a peer that was created in error, as long as no payments have been exchanged. Active peer management helps maintain seamless payment flows and keeps your Rafiki instance up to date with operational changes.
+
+### Edit a peer
+
+Editing a peer allows you to adjust peering configurations or address any changes communicated by the peer. Some examples include updating new endpoints or tokens, technical settings like the maximum packet amount, or peer liquidity thresholds.
+
+In this example we are going to update the peer we just created. Rather than change any of the peering details, we can add some optional details that we didn't include when we created the peer. We will define the `maxPacketAmount` and the `liquidityThreshold`.
+
+#### `updatePeer`
+
+<CodeBlock title="Example updatePeer Mutation">
+
+```graphql
+mutation UpdatePeer($input: UpdatePeerInput!) {
+  updatePeer(input: $input) {
+    peer {
+      id
+      name
+      http {
+        outgoing {
+          authToken
+          endpoint
+        }
+      }
+      maxPacketAmount
+      liquidityThreshold
+    }
+  }
+}
+```
+
+</CodeBlock>
+
+##### Example
+
+The input object for the update operation only requires that the `id` is present. All other variables are optional. For this example we will include the required `id` variable, as well as the optional variables of the fields we wish to update. In this case, `maxPacketAmount` and `liquidityThreshold`.
+
+<CodeBlock title='Example JSON request'>
+
+```json
+{
+  "input": {
+    "id": "480ef339-7842-4501-a905-923fc1339cef",
+    "maxPacketAmount": 1000,
+    "liquidityThreshold": 100
+    }
+}
+```
+
+</CodeBlock>
+
+<div class="overflow-table">
+
+| Variable                   | Description                                                                                                   | Required |
+| -------------------------- | ------------------------------------------------------------------------------------------------------------- | -------- |
+| `id`                       | The ID of the peer you wish to update.                                                                        | Y        |
+| `maxPacketAmount`          | Maximum packet amount that the peer accepts.                                                                  | N        |
+| `liquidityThreshold`       | A webhook event will notify the Account Servicing Entity if peer liquidity falls below this new value.        | N        |
```

Comment:
While the example only uses `maxPacketAmount` and `liquidityThreshold` and identifies them as two possible optional fields, I wonder if we should add all the possible fields to the table so that the reader doesn't assume those are the only two.  Alternatively, we could remove the table.

Reply:
I'm not dismissing this feedback, but I think that the reader could look at the API reference to see that these are not the only possible fields. In https://github.com/interledger/rafiki/issues/3090, we are going to update these code examples to link to a mutation's input object to hopefully help bridge that gap. Our code examples shouldn't have to account for all fields available.